src/v3/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -22,9 +22,8 @@ import Image from '../Image';
 
 // TODO: maybe extract to util class if used reused
 const shouldRenderAuthCoin = (props?: AuthCoinProps): boolean => {
-  const authCoinConfiguration = getAuthCoinConfiguration();
   const authCoinConfigByAuthKey = props?.authenticatorKey
-    && authCoinConfiguration[props?.authenticatorKey];
+    && getAuthCoinConfiguration()[props.authenticatorKey];
```

Comment:
Not a necessary change, just small optimisation.
Noticed that for pages that don't have auth coin there will be a lot of unnecessary `loc` calls inside `getAuthCoinConfiguration()`